components/consent.js

```diff
@@ -7,11 +7,12 @@ function Consent({handleConsent}) {
       <p>Le questionnaire que vous vous apprêtez à remplir n’a pas de valeur d’avis médical. Il sert uniquement à vous orienter en fonction de votre état de santé et des symptômes que vous indiquez.</p>
       <p>Vos réponses ne serviront qu’à des fins de recherche pour mieux comprendre l’épidémie de COVID 19.</p>
       <p>Aucune information permettant de vous identifier, directement ou indirectement, ne vous sera demandée.</p>
-
+      <br/>
+      <p>L’algorithme d’orientation COVID 19 est mis en œuvre conformément à la <a target="new" title="Accéder à la documentation dans un nouvel onglet" href="https://delegation-numerique-en-sante.github.io/covid19-algorithme-orientation/index.html">documentation officielle</a> publiée par le ministère des Solidarités et de la Santé disponible.</p>
```

Comment:
Ajouter `target='_top'` au lien pour que la page be s’ouvre pas dans l’iframe.

Reply:
on reconnait les anciens 😂

Reply:
Fait !